slice/Ice/SlicDefinitions.ice

```diff
@@ -19,41 +19,25 @@ module Ice
     // These definitions help with the encoding of Slic frames.
     module Slic
     {
-        /// The body of a Slic initialize frame.
-        /// TODO: should we use tagged data members instead for parameters? If we do, we can no longer use structs
-        /// however
+        /// The keys for supported Slic transport parameters. Slic transport parameters are encoded with the same
+        /// encoding as binary contexts.
+        enum ParameterKey : int
+        {
+            MaxBidirectionalStreams = 0,
+            MaxUnidirectionalStreams = 1,
+            IdleTimeout = 3,
+        }
+
+        /// The header of the Slic initialize frame body. This header is followed by transport parameters encoded
```

Comment:
I don't find transport parameters very clear - the transport here is Slic, isn't it?
Maybe "connection parameters" or "Slic connection parameters"?

Reply:
Yes, parameters for the Slic transport. It could also be Slic protocol parameters... or connection parameters, to me it's pretty much the same. I'll change it to Slic connection parameters.